src/Entity/Course.php

```diff
@@ -112,7 +112,8 @@ public function __construct(
         string $description,
         float $price,
         float $duration,
-        string $content
+        string $content,
+        string $summary
```

Comment:
Toto by mal byť objekt. Alebo sa v konštruktore zo stringu musí vyrobiť objekt (lenže ten potrebuje 2 stringy).

Každopádne ak si to chceš uľahčiť tak zahoď tú OneToOne väzbu a pridaj si tam stĺpčeky... imho je to pekné, ale zbytočne komplikované.

Reply:
opraveno